### PR TITLE
Pod Security Policy doc improvements

### PR DESCRIPTION
**Pull request by php-coder**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3010)

### CODE REVIEW

File: docs/user-guide/pod-security-policy/psp.yaml [hunk 1]
```diff
@@ -1,28 +1,18 @@
...
-      "HostPorts": [
-          {
-          "min": 8000,
-          "max": 8080
-          }
-      ],
-      "volumes": ["*"]
-  }
-}
+apiVersion: extensions/v1beta1
```
Comment (pweil-) [inline docs/user-guide/pod-security-policy/psp.yaml]: hmm, just saw this this morning too https://github.com/kubernetes/kubernetes.github.io/pull/3003  so probably need to make sure we're not stepping on toes between the two

Comment (php-coder) [inline docs/user-guide/pod-security-policy/psp.yaml]: I believe that we should use YAML :)

**Comment by php-coder** [comment]
> Do you want to include more info on enabling of the admission controller as discussed in the recent issues?

I'd prefer to solve this in a separate PR if you don't mind.

**Comment by pweil-** [comment]
> I'd prefer to solve this in a separate PR if you don't mind.

yep, fine by me

File: docs/user-guide/pod-security-policy/psp.yaml (hunk 1 above)
Comment (pweil-) [inline docs/user-guide/pod-security-policy/psp.yaml]: +1 to yaml.  Added a comment to the other PR.

**Comment by chenopis** [comment]
@pweil- IMHO we should move forward w/ YAML. Assuming that, is this Tech LGTM?

**Comment by pweil-** [comment]
> Assuming that, is this Tech LGTM?

yes, LGTM.  

@php-coder please squash

**Comment by chenopis** [comment]
I can squash and merge.

**Comment by chenopis** [comment]
@php-coder Yes, thanks for catching that. Any sample YAML should be in a separate file so that it can be tested, so please add it to `test/examples_test.go` in another PR and ping me for review. Cheers.

**Comment by php-coder** [comment]
> please add it to test/examples_test.go in another PR and ping me for review

@chenopis I've created https://github.com/kubernetes/kubernetes.github.io/pull/3100

**Comment by chenopis** [comment]
@php-coder Great! Thank you!

**Pull request merged**